docs/DeveloperGuide.adoc

```diff
@@ -292,23 +292,75 @@ Refer to the guide <<DevOps#, here>>.
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="90%",cols="20%,<25%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |diabetic patient who has different options on medical care |know exactly how much I am spending on medication and consultation |know which hospitals to seek medical care from
 
-|`* * *` |user |add a new person |
+|`* * *` |very busy diabetic |use a flexible calendar system that can account for updates | easily make changes to appointments that I have to change often due to other commitments
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |diabetic |keep track of my medical expenses |better manage my finance
```

Comment:
This seems to repeat the 3rd last part of your user stories.
The one with "a diabetic patient who has many medical receipts - and is not very good at mathematics"